components/constellation/constellation.rs

```diff
@@ -2300,7 +2312,13 @@ where
         let url = ServoUrl::parse("about:blank").expect("infallible");
 
         // TODO: Referrer?
-        let load_data = LoadData::new(url.clone(), Some(parent_pipeline_id), None, None);
+        let load_data = LoadData::new(
+            LoadOrigin::Script(url.origin()),
```

Comment:
Why is this the origin associated with `url` (which is always going to be a fresh opaque origin, since `url` is `about:blank`)?

Reply:
Thanks, I switched to using the `LoadOrigin` that is sent as part of the `IFrameLoadInfo`.